federation/src/graphql/api/1_1/resolver/TestResolver.ts

```diff
@@ -1,8 +1,10 @@
-import { Field, ObjectType, Query, Resolver } from 'type-graphql'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { Query, Resolver } from 'type-graphql'
 import { federationLogger as logger } from '@/server/logger'
 import { GetTestApiResult } from '../../GetTestApiResult'
 
 @Resolver()
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Same question as the unit-test file

Reply:
here i have no answer why, but my IDE give this hint...